src/biogeophys/TemperatureType.F90

```diff
@@ -647,11 +647,11 @@ subroutine InitCold(this, bounds, &
     ! !USES:
     use shr_kind_mod   , only : r8 => shr_kind_r8
     use shr_const_mod  , only : SHR_CONST_TKFRZ
-    use clm_varcon     , only : denice, denh2o, sb
-    use landunit_varcon, only : istwet, istsoil, istdlak, istice
+    use clm_varcon     , only : denice, denh2o, sb !what's sb? how it's never used here
```

Comment:
If sb isn't used here, we should remove it. If it is used, it should fail in the build so you should see quickly whether you can do that or not. Thanks for pointing this out, by the way.

Reply:
I was able to build and run after removing `sb`.